terminus-ui/radio-group/src/radio-group.component.spec.ts

```diff
@@ -135,7 +139,9 @@ describe('TsRadioGroupComponent', function() {
 
 
     test(`should throw an error in dev mode when passed a value that is not a function`, () => {
-      expect(() => {component.formatModelValueFn = 3 as any; })
+      expect(() => {
+        component.formatModelValueFn = 3 as any;
```

Comment:
Spec & mock files have less strict rules since we often need to do 'hacky' stuff to make sure our components are fully tested. When I finish integrating the new rules into ngx-tools I can truly 'finalize' the rulesets and we can walk through them as a team before they are rolled out to the rest of the org.

You can see here which rules are relaxed for spec/mock files: https://github.com/GetTerminus/tslint-config-frontend/blob/release/testing.js